test/e2e/framework/scaletest/validate-nodes.go

```diff
@@ -0,0 +1,65 @@
+package scaletest
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+const (
+	defaultTimeoutSeconds = 300
+)
+
+type ValidateNumOfNodes struct {
+	NumNodesRequired   int
+	Label              map[string]string
+	KubeConfigFilePath string
+}
+
+// Useful when wanting to do parameter checking, for example
```

Comment:
this comment is a bit redundant when copy pasted to every struct :)

Reply:
This comment comes from the interface. I use a plugin to automatically create the methods from the interface that I want to implement, so it also gets the comment from it :)